salt/search/config/srv-search-config.php

```diff
@@ -25,7 +25,6 @@
         'endpoint' => '{{ pillar.search.aws.endpoint }}',
         {% endif %}
     ],
-    'feature_rds' => {% if pillar.search.feature_rds %}true{% else %}false {% endif %},
```

Comment:
would rather re-add this as `=> true`, rather than relying on the pillar, if we want to reinstate this field because it's suspected to be breaking `search--ci`